esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1381,7 +1381,8 @@ void Esdl2Response::processChildNamedResponse(Esdl2TransformerContext &ctx, cons
                 if (ctx.do_output_ns)
                 {
                     ctx.writer->outputBeginNested(out_name, true);
-                    ctx.writer->outputXmlns(out_name, ctx.ns.str());
```

Comment:
Not sure what the best answer is here, but this change breaks non-scapps services that could depend on this implementation.

Reply:
This is a change but Tony called it out as a bug, and we shouldn't return it like that.

Reply:
Based on offline conversation, this is OK as-is.